web/src/components/Prices/HollaexToken.js

```diff
@@ -0,0 +1,570 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Select } from 'antd';
+
+import { StarFilled, StarOutlined } from '@ant-design/icons';
+import math from 'mathjs';
+import STRINGS from 'config/localizedStrings';
+import { BASE_CURRENCY, DEFAULT_COIN_DATA } from 'config/constants';
+import { formatPercentage } from 'utils/currency';
+import { Button, EditWrapper, IconTitle, Image, PriceChange } from 'components';
+import SparkLine from 'containers/TradeTabs/components/SparkLine';
+import { getSparklines } from 'actions/chartAction';
+import withConfig from 'components/ConfigProvider/withConfig';
+import { isLoggedIn } from 'utils/token';
+import { addToFavourites, removeFromFavourites } from 'actions/appActions';
+import { handleReplace } from './Utils';
+
+const PAIR_2_COINS = ['xht', 'btc', 'eth'];
+const ACTIVITY_BLOCKCHAIN =
+	'https://etherscan.io/token/0xD3c625F54dec647DB8780dBBe0E880eF21BA4329';
+
+const Hollaextoken = (props) => {
+	const {
+		pairs,
+		tickers,
+		coins,
+		icons: ICONS,
+		router,
+		available_balance,
+	} = props;
+	const currentPath = window?.location?.pathname.split('/');
```

Comment:
The token parameter is directly available in the router object. We should drop both of these lines